synchronization/src/main/java/de/cyface/synchronization/SyncAdapter.kt

```diff
@@ -177,17 +179,21 @@ class SyncAdapter private constructor(
                             for (listener in progressListener) {
                                 listener.onProgress(total.toFloat(), measurement.id)
                             }
-                        }, jwtAuthToken
+                        }
+                    }
+                    val result = syncPerformer.sendData(
+                        http, syncResult, endPointUrl, metaData,
+                        compressedTransferTempFile!!, processListener, jwtAuthToken
```

Comment:
If there is a line break, please put each parameter on its own line. This will ease future reviewes and is the recommended style for Swift and Kotlin Code.